web/profiles/custom/yalesites_profile/config/sync/core.extension.yml

```diff
@@ -82,5 +82,5 @@ theme:
   atomic: 0
   claro: 0
   gin: 0
-  bartik: 0
```

Comment:
I could have sworn this was just added as a dependency for the gin theme, but I can not find a reason in the code to backup my assumption. So I suppose it's okay that bartik is going away. Not sure why it was added in the first place.

Reply:
Gin uses Claro as a base theme, so I don't think Bartik is needed...